services/Directory/data/FilterList.json

```diff
@@ -12958,6 +12958,7 @@
     "emailAddress": "imreeil42@gmail.com",
     "homeUrl": "https://github.com/DandelionSprout/adfilt",
     "id": 1929,
+    "isDeleted": true,
```

Comment:
FYI, this doesn't work currently, but it's on the to-fix list. https://github.com/collinbarrett/FilterLists/issues/713#issuecomment-696444282